app/Domain/Clients/Controllers/NewClient.php

```diff
@@ -49,23 +75,23 @@ public function run()
                     'internet' => '',
                     'email' => '',
                 ];
-
-                if (isset($_POST['save']) === true) {
-                    $values = [
-                        'name' => ($_POST['name']),
-                        'street' => ($_POST['street']),
-                        'zip' => ($_POST['zip']),
-                        'city' => ($_POST['city']),
-                        'state' => ($_POST['state']),
-                        'country' => ($_POST['country']),
-                        'phone' => ($_POST['phone']),
-                        'internet' => ($_POST['internet']),
-                        'email' => ($_POST['email']),
+                
+                if (isset($params['save']) === true) {
+                    $values = (object) [
```

Comment:
Same as above. Please create a model and then map to the model directly. We don't do that everywhere but you can take at the tickets model. In there we use the constructor to map arrays to the model.